images.yaml

```diff
@@ -582,6 +591,10 @@
   tags:
   - sha: 1fad3f247a8edeceead03230d833fcb4c3935a3a57b83787690f15f7daa6b59b
     tag: v5.1.0
+- name: quay.io/reactiveops/ci-images
```

Comment:
What is this?

Reply:
This image is used to generate the certificates for the admission controller.

Reply:
This image is used by VPA to generate certificates for the admission controller and is used internally by the helm chart for testing